src/main/java/ja/workshops/spring/części/konfiguracja/xml/c_mvc/RepozytoriumSamochodów.java

```diff
@@ -0,0 +1,40 @@
+package ja.workshops.spring.części.konfiguracja.xml.c_mvc;
+
+import ja.workshops.spring.części.konfiguracja.xml.c_mvc.silnik.Benzyna;
+import ja.workshops.spring.części.konfiguracja.xml.c_mvc.silnik.OlejNapędowy;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Repozytorium samochodów. Ziarno obrazujące warstwę, w której mamy dostęp do lokalnej bazy danych.
```

Comment:
Bez `lokalnej`, może imitacja, albo `bazy danych` wyróżnić cudzysłowiem albo zaznaczyć, że to nasza wewnątrz programu baza. Lokalna znaczy "na tym samym serwerze" lub w sieci LAN.